module/VuFind/sql/migrations/pgsql/9.0/001-add-search-index.sql

```diff
@@ -0,0 +1,5 @@
+--
+-- Modifications to table `search`
+--
+
+CREATE INDEX search_created_saved_idx ON search (created, saved);
```

Comment:
Is there a reason that the index name in the migration is different from the index name in the pgsql.sql file?

Reply:
Apart from an unsaved file in editor, no, not really. :) Thanks for catching that!